src/notation/view/widgets/editstyle.cpp

```diff
@@ -68,6 +63,17 @@ static void fillDirectionComboBox(QComboBox* comboBox)
     comboBox->addItem(Ms::toUserString(Ms::Direction::DOWN), QVariant::fromValue<Ms::Direction>(Ms::Direction::DOWN));
 }
 
+template<class T>
+void EditStyle::setWidgetIcon(T* widget, IconCode::Code code)
```

Comment:
This function looks quite useful. It seems that we can reuse it in other widget dialogs. I suggest creating a new class with this function as a static method.

Reply:
Yeah, that was what I was planning to do. Indeed, we also need it for #9508, which I will also fix.